File: web3rpc/rpc-specs/code-samples/python/eth/transaction/getTransactionByBlockHashAndIndex.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

blockHash = "0x451cafae98d61b7458b5cef54402830941432278184453e3ca490eb687317e68"
transactionIndexPosition = "0x0"

w3 = Web3(Web3.HTTPProvider(host))
eth_response = w3.eth.get_transaction_by_block_hash_and_index(blockHash, transactionIndexPosition)

print(eth_response)

File: web3rpc/rpc-specs/code-samples/javascript/eth/transaction/getTransactionByBlockHashAndIndex.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

  const blockHash =
    "0xc9dbfbab67e9a0508bcb3f95ae408023668cef431b805592781a821926715b8a";
  const index = "0x0";
  provider.eth
    .getTransactionByBlockHashAndIndex(
      blockHash,
      index,
      {},
      (err, data, response) => {},
    )
    .then((data) => {
      console.log(data);
    });
})();
